src/index.scss

```diff
@@ -0,0 +1,9 @@
+@import 'carbon-components/scss/globals/scss/styles.scss';
+
+main {
+  display: block;
+}
+
+$feature-flags: (
+  ui-shell: true,
+);
```

Comment:
This definition should come before the carbon styles import.

Reply:
Hi Kelly, moved the definition as requested